frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.reducer.ts

```diff
@@ -114,7 +117,27 @@ const reducer = createReducer(
   on(FeatureFlagsActions.actionSetActiveDetailsTabIndex, (state, { activeDetailsTabIndex }) => ({
     ...state,
     activeDetailsTabIndex,
-  }))
+  })),
+  on(FeatureFlagsActions.actionAddFeatureFlagInclusionList, (state) => ({
+    ...state,
+    isLoadingUpsertPrivateSegmentList: true,
+  })),
+  on(FeatureFlagsActions.actionUpsertFeatureFlagInclusionListSuccess, (state, { listResponse }) => {
+    const { featureFlag } = listResponse;
+    const existingFlag = state.entities[featureFlag?.id];
+
+    return adapter.updateOne(
+      {
+        id: featureFlag?.id,
+        changes: { featureFlagSegmentInclusion: [listResponse, ...existingFlag.featureFlagSegmentInclusion] },
+      },
+      { ...state }
+    );
+    return adapter.upsertOne(featureFlag, { ...state, isLoadingUpsertPrivateSegmentList: false });
+  }),
+  on(FeatureFlagsActions.actionUpsertFeatureFlagInclusionListFailure, (state) => {
```

Comment:
@danoswaltCL It's OK to remove this line right? This is not called because of the `return adapter.updateOne()` above it.

Reply:
oops yes. I'm surprised some kind of linting error doesn't happen here since it's unreachable code.